sunpy/timeseries/timeseriesbase.py

```diff
@@ -445,6 +445,33 @@ def concatenate(self, others, same_source=False, **kwargs):
         object._sanitize_units()
         return object
 
+    def reindex(self, index, **kwargs):
+        """
+        Returns a new time series with the chosen index.
```

Comment:
```suggestion
        Returns a new time series with a new index.
        
        By default values on the new time series are filled using a
        nearest valid observation method. See `~pandas.DataFrame.reindex`
        for the different re-indexing options available.
```

Reply:
I believe the default method for `reindex` is `None`, shall I add another param to default it to `nearest`?